draft-ietf-sframe-enc.md

```diff
@@ -1557,3 +1693,85 @@ The other values in the test vector are intermediate values provided to
 facilitate debugging of test failures.
 
 {::include test-vectors/sframe.md}
+
+# Acknowledgements
+{: numbered="false"}
+
+The authors wish to specially thank {{{Dr. Alex Gouaillard}}} as one of the early
+contributors to the document. His passion and energy were key to the design and
+development of SFrame.
+
+<!--[rfced] Terminology.
+
+a) FYI, we have added expansions for abbreviations upon first use per Section 3.6 of RFC 7322 ("RFC Style Guide"). Please review each expansion in the document carefully to ensure correctness.
+
+b) Please help us expand or define the following term:
+
+   SVC
+
+c) May we make the use of the acronym KID consistent?
+
+   Key ID (13) / KID (85)
+
+d) Please help us make the capitalization of the following terms consistent
+
+   counter (22) / Counter (9) - We note that Key ID is capitalized.
+                                Should CTR be used instead?
+
+e) Please review the use of fixed-width format for the following terms and let us know if any updates are necessary:
+
+   context, when referring to a value
+   CTR
+   key, when referring to input
+   KID
+   nonce
```

Comment:
TODO(RLB): Review these for consistency.

Reply:
* CTR and KID should never be fixed-width.  It does not appear they ever are, outside of pseudocode.
* "context" should only be fixed width in the context of the MLS section, where it refers to a specific value.
* "key" and "nonce" look to be correctly only fixed-width in the context of the test vectors.